build.sh

```diff
@@ -302,20 +304,16 @@ fi
 ######################################
 
 kill_object_server() {
-    (pgrep -f realm-object-server || true) | while read pid; do
-        kill $pid 2>/dev/null
-    done
+# Based on build.sh conventions we always run ROS from a path ending in 'ros/bin/ros'.
+    pid=$(ps ax | grep "ros/bin/ros start$" | sed -e 's/^[[:space:]]*//' | awk '{ print $1 }')
+    kill $pid 2>/dev/null
 }
 
 download_object_server() {
-    local archive_name="realm-object-server-bundled_node_darwin-developer-$REALM_OBJECT_SERVER_VERSION.tar.gz"
-    /usr/bin/curl -L -O "https://static.realm.io/downloads/object-server/$archive_name"
-    rm -rf sync
-    mkdir sync
-    tar xf $archive_name -C sync
-    rm $archive_name
-    cp Configuration/object-server-config.yml sync/object-server/configuration.yml
-    touch "sync/object-server/do_not_open_browser"
+    rm -rf ./test-ros-instance
+    mkdir ./test-ros-instance
+    mkdir ./test-ros-instance/ros
```

Comment:
The `./` are unnecessary here.

The two `mkdir` invocations can become `mkdir -p test-ros-instance/ros`.